cmd/nodeman.go

```
@@ -0,0 +1,721 @@
+	if respPayload.MqttBroker != "" {
+        cfg.MqttBroker = respPayload.MqttBroker
+	}
+	if respPayload.AggregatesUrl != "" {
+        cfg.AggregatesUrl = respPayload.AggregatesUrl
+	}
+
+    tmlSources, err := template.New("sources").Parse(CFG_TML_POP_SOURCES)
+    if err != nil {
+        panic(err)
+    }
+	fhSources, err := os.OpenFile(sourcesFilename, os.O_RDWR|os.O_CREATE|os.O_EXCL, 0o600)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)